text/0000-location.md

```diff
@@ -0,0 +1,409 @@
+(Organizations and agents are defined by the Pike smart contract.)
+
+If the Grid setting grid.location.allow_delete is set to false, this transaction
+is invalid. The default value for grid.location.allow_delete is true. This 
+setting is stored using the Sawtooth Settings smart contract, more information
+can be found here.
+
+Validation requirements:
+- If a location with location_id does not exist the transaction is invalid.
+- The signer of the transaction must be an agent in the Pike state and must 
+belong to an organization in Pike state, otherwise the transaction is invalid.
+- The owner in the location must match the organization that the agent belongs 
+to, otherwise the transaction is invalid.
+- The agent must have the permission �can_delete_location� for the organization
+otherwise the transaction is invalid.
+
+The inputs for LocationDeleteAction must include:
+- Grid address of the Agent submitting the transaction
+- Grid address of the Organization the Location is being deleted for Grid 
+address of the Location to be deleted
+
+The outputs for LocationDeleteAction must include:
+- Grid address of the Location to be deleted
+
+### Defined GS1 Properties
+
+This Location RFC defines required and optional attributes for GS1 Locations 
+(see table below). This implementation does not include attribution for specific
+industries, financial/tax account information, a physical location extension, or
+digital locations and should be implemented with supplemental RFC(s).
+
+**_REQUIRED FIELDS_**
+|GS1 Common Name|GS1 Attribute Name|Description|Example|Type|Min|Max|
+|---------------|------------------|-----------|-------|----|---|---|
+|Location Name 1|locationName|The name of the facility being described.|"Sunny Fresh Foods"|STRING|1|80|
+|Description|locationDescription|Free text, 178 characters.|"A Cargill production facility dedicated to serving high-quality egg products across various markets."|STRING| | |
+|Location Type|locationType|Multiple types allowed. All Suppliers: Org Entity, Order From, Remit To, Ship To. Healthcare Providers: Bill To, Deliver To, Order By, Order From, Org Entity, Paid By, Recall, Remit To, Ship From, Ship To.|"Ship From"|ENUM|7|48|
+|Address Line 1|addressLine1|The primary street address for your location. The USPS address is validated if Country = United States.| "206 W 4th Street"|STRING|1|80|
+|City|city|Name of the city of your location. The USPS address is validated if Country = United States.|"Monticello"|STRING|1|35|
+|State or Region|stateOrRegion|The state, province, or region using the standard two-letter abbreviation specified in ISO 3166-2:1998 country subdivision code [16].|"MN"|STRING|1|3|
+|Postal Code|postalCode|The ZIP or other postal code.|"55362-8524"|STRING|1|10|
+|Country|country|Country of your location. Spell out, do not use abbreviations.|"United States"|STRING|2|80|
+|Latitude|latitude|(for fields)|"44.986656"|LAT| | |
+|Longitude|longitude|(for fields)|"-93.258133"|LONG| | |
```

Comment:
The current property definition supports NUMBER, STRING, or LAT_LONG; since both of these are required, it probably makes sense to store them together as LAT_LONG.